common/djangoapps/student/models.py

```diff
@@ -156,6 +156,20 @@ def user_by_anonymous_id(uid):
         return None
 
 
+def user_can_skip_entrance_exam(user, course_key):
+    """
+    Return True if given user can skip entrance exam for given course otherwise False.
+    """
+    can_skip = False
+    if settings.FEATURES.get('MILESTONES_APP', False):
```

Comment:
Should we be checking for the milestones app or the entrance exams feature here?  Seems like it should be entrance exams to me.